west.yml

```diff
@@ -52,7 +52,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 8a249246bc2add52db9d1b9ee5db111e0ccda4c3
+      revision: aa1dc9c1c14d5bfa420df329b43b2b297793df5d
```

Comment:
I guess this will be removed when a rebase is done?

Reply:
No, this is needed. We're pulling upstream changes.

Reply:
SHA was updated in https://github.com/nrfconnect/sdk-nrf/pull/9073 to a more recent revision, please drop this change.

Reply:
done